dags/resources/stages/parse/table_definitions/aave/Proxy_event_ReserveInitialized.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
@ivigamberdiev , we might need to create a PR with just this change first so we can reference it on all the other json files. 

Will also be able to make sure it covers all those addresses then :)